probes/wasCreatedInLast90Days/def.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: wasCreatedInLast90Days
+short: Checks whether the project was created in the last 90 days.
+motivation: >
+  When Scorecard checks the activity of a project in the last 90 days, the project may not have been created before the last 90 days. As such, Scorecard cannot give an accurate score. This probe helps Scorecard assess whether it can give an accurrate score when checking the project activity in the last 90 days.
+implementation: >
+  The implementation checks the creation date is within the last 90 days.
+outcome:
+  - If the project was created within the last 90 days, the outocome is OutcomeNegative (0).
+  - If the project was created before the last 90 days, the outocome is OutcomePositive (1).
```

Comment:
i think this has the similar confusion to archived.

Reply:
I renamed the probe to `notCreatedInLast90Days` here: https://github.com/ossf/scorecard/pull/3507/commits/9fa6527c9b1ca34ecb26c71271d17fdf72ce3457. Now, `OutcomePositive` means "Yes, the project is not created in the last 90 days".